tests/test_info.py

```diff
@@ -195,3 +195,9 @@ def test_is_draft_pull_request(fake_github, title):
     # No matter what the title, a pr is Draft if it says it is.
     pr = fake_github.make_pull_request(title=title, draft=True)
     assert is_draft_pull_request(pr.as_json())
+
+def test_check_csv_priority():
+    people = get_people_file()
+    user = 'Carlos-Muniz'
+    # This user exists in the yaml but not in the csv, and should not be in people
+    assert people.get(user) == None
```

Comment:
The last line is missing a newline.  Is your editor configured to read .editorconfig files? It should be, to automatically enforce some rules like this.

Reply:
Test and implementation has been added. 
And I've added the .editorconfig functionality to my editor